web/ViewModels/LandlordProfileModel.cs

```diff
@@ -50,7 +57,9 @@ public static LandlordProfileModel FromDbModel(LandlordDbModel landlord)
             Title = landlord.Title,
             LandlordStatus = landlord.LandlordStatus,
             LandlordProvidedCharterStatus = landlord.LandlordProvidedCharterStatus,
-            CharterApproved = landlord.CharterApproved
+            CharterApproved = landlord.CharterApproved,
+            CurrentUserIsAdmin = user.IsAdmin,
+            CurrentUserId = user.Id

```

Comment:
CurrentUserIsAdmin will slightly affect the displayed profile (things like the 'Approve Landlord' button will only appear if the current user is an admin). As far as I know, incorporating this property into the LandlordProfileModel is the easiest (if not only) way send this information to the Profile page and thus adjust the view accordingly.

CurrentUserId *was* included so that, if the user is an admin and they approve the landlord, the id of the admin who performs this action can be saved to the database (as specified in the ticket). However, having just re-checked the code, this doesn't actually seem to be needed - I think when I first made changes to the Profile Model, I forgot that it was possible to GetCurrentUser() between pressing the 'Approve Landlord' button and the database changes occurring. I've deleted this property now.

Reply:
In this specific case you could do `@if (User.IsInRole("Admin"))` in the Razor view.

Probably the "right" way to do things like this in general where you're adding extra page-specific data would be to make a new "LandlordProfilePageViewModel" that includes this model as a property along with "CurrentUserIsAdmin".

If we end up adding several extra properties here then it's worth changing, but otherwise I wouldn't worry